.vscode/tasks.json

```diff
@@ -43,6 +43,34 @@
         "reveal": "always",
         "panel": "dedicated"
       }
+    },
+    {
+      "label": "Functional tests - all",
+      "detail": "Runs the functional tests against the web app on port 5000",
+      "type": "shell",
+      "command": "npm test",
+      "options": {
+        "cwd": "${workspaceFolder}/functional-tests"
+      },
+      "group": "test",
+      "presentation": {
+        "reveal": "always",
+        "panel": "dedicated"
+      }
+    },
+    {
+      "label": "Functional tests - current feature",
+      "detail": "Runs the functional tests for the currently open feature file",
+      "type": "shell",
+      "command": "npm test -- --spec ${fileBasenameNoExtension}",
```

Comment:
I dont know npm very well, wondered why there are two sets of --

Reply:
See https://docs.npmjs.com/cli/run-script#description - it's for passing in parameters to child scripts

Reply:
On a vagutely related note f you get the new cross platform powershell (7) it colour codes tasks nicely so you can see args etc which is kinda nice